test/contributionreward.js

```diff
@@ -37,36 +37,33 @@ const checkRedeemedPeriodsLeft = async function(
                                             EtherPeriod,
                                             ExternalTokenPeriod
                                             ) {
-    assert.equal(await testSetup.contributionReward.getPeriodsToPay(proposalId,testSetup.org.avatar.address,web3.utils.toBN(0)),ReputationPeriod);
-    assert.equal(await testSetup.contributionReward.getPeriodsToPay(proposalId,testSetup.org.avatar.address,web3.utils.toBN(1)),nativeTokenPeriod);
-    assert.equal(await testSetup.contributionReward.getPeriodsToPay(proposalId,testSetup.org.avatar.address,web3.utils.toBN(2)),EtherPeriod);
-    assert.equal(await testSetup.contributionReward.getPeriodsToPay(proposalId, testSetup.org.avatar.address,web3.utils.toBN(3)),ExternalTokenPeriod);
+    assert.equal(await testSetup.contributionReward.getPeriodsToPay(proposalId,web3.utils.toBN(0)),ReputationPeriod);
+    assert.equal(await testSetup.contributionReward.getPeriodsToPay(proposalId,web3.utils.toBN(1)),nativeTokenPeriod);
+    assert.equal(await testSetup.contributionReward.getPeriodsToPay(proposalId,web3.utils.toBN(2)),EtherPeriod);
+    assert.equal(await testSetup.contributionReward.getPeriodsToPay(proposalId,web3.utils.toBN(3)),ExternalTokenPeriod);
 };
 
 const setupContributionRewardParams = async function(
```

Comment:
Maybe rename to `setupContributionReward`?

Reply:
this contract is non-universal